community/sample-apps/whatsapp-clone-typescript-react/hasura/migrations/whatsappdb.up.sql

```diff
@@ -0,0 +1,427 @@
+--
```

Comment:
@praveenweb Can you check if `migrate apply` works on this file?

Reply:
Updated the migration file and metadata to make it work with migrate apply.